hangb.rb

```diff
@@ -0,0 +1,262 @@
+      puts """
+                                  _-_
+                         ____.---'---`---.____
+                          \\----._________.----/
+                          /  /    `-_-'
+              __,--`.`-'..'-_
+             /____          ||
+                  `--.____,-'
+
+      """.yellow
+    elsif @wrong_count == 4
+      puts """
+                                  _-_
+                         ____.---'---`---.____
+                          \\----._________.----/
+                          /  /    `-_-'
+
+
+
+
+
+      """.magenta
+    elsif @wrong_count == 5
+      puts """
+                                  _-_
+                         ____.---'---`---.____
+                          \\----._________.----/
+                                  `-_-'
+
+
+
+
+
+      """.red
+    else
+      puts "Sorry,WARP CORE BREACH! You lose!".red.blink
+      exit
+    end
+
+  end
+
+# m: " Here's the method definition for @answer_line - It is equal to an empty array.
+#    When you shuffle.pop the "answer" out, create a line of empty underscores exactly the same
+#    length as the "answer." Put all the underscores in the @answer_line array together as a string
+#    - that's what ".join" means. "return" that after you create it, so I (and the user) can see it.
+# U: Word.
+
+  def answer_line
+    @answer_line = []
+    (@answer.length).times do |ans|
+      @answer_line.push(" _ ")
+    end
+    puts @answer_line.join
+    return @answer_line
+  end
+
+
+# ------------------
+# Check answer
+
+# m: "U, girl. Here's the method definition for get_guess. This is how I think it would be easiest for you
+#      to return an answer to the user..."
+# U: Aiite, go.
+# m: " Okay,
+#     1st - until that answer line equals the answer from shuffle.pop: draw the board, and keep
+#           displaying the answer line so they'll know how awesome they're doing. Politely ask
+#           for a letter while those conditions are true as well.  
+#     2nd - If someone/thing tries to give you more than one letter, scold them!
+#     3rd - When the user gives you a letter as a guess, evaluate that input against @answer.
+#           That's an array though, so you'll have to .split the empty spaces from the input 
+#           to make sure it looks exactly like the letters of the chosen @answer.
+#     4th - accept the input letter(s) in lower or uppercase; but since all the answers are written in upper-
+#           case, turn the user input into uppercase before you evaluate it against the @answer."
+#     Still with me?
+#  U: yup.
+#  m: "awesome, let me break down this conditional loop for you below."
+ 
+
+
+  def get_guess
+
+    until @answer_line.join == @answer
+
+      puts "Guess a letter, please:"
+      input = gets.chomp.upcase
+
+      while input.length != 1
+        puts "You're better than this. Please enter ONE letter, thanks."
+        input = gets.chomp.upcase
+      end
+
+      letters = @answer.split("")
```

Comment:
Since you're in a loop, you'd end up doing the split every time you loop. You could define letters before the loop, so it'd only happen once.